### PR TITLE
#500 - Linux Metric - Memory: evaluate free memory from MemFree when …

### PR DESCRIPTION
**Pull request by NielkS**
…MemAvailable not mentioned in /proc/meminfo

### CODE REVIEW

**Comment by beniwohli** [comment]
jenkins test this please

**Comment by NielkS** [comment]
HI @beniwohli ,
Not sure why the build fails - maybe related to my force-push ?
Please let me know if I should create another PR with "clean" commit history.

File: elasticapm/metrics/sets/cpu_linux.py [hunk 1]
```diff
@@ -70,7 +70,9 @@ def collect(self):
             delta = {k: new[k] - prev[k] for k in new.keys()}
             cpu_usage_ratio = delta["cpu_usage"] / delta["cpu_total"]
             self.gauge("system.cpu.total.norm.pct").val = cpu_usage_ratio
-            self.gauge("system.memory.actual.free").val = new["mem_available"]
+            # mem_available not present in linux before kernel 3.14 - fallback to mem_free if not present - see #500
```
Comment (axw) [inline elasticapm/metrics/sets/cpu_linux.py]: At least some of the other agents are are using `Free + Buffers + Cached` as the result, when `MemAvailable` is not present. Would you please change this to do the same for consistency?

For example, this is the relevant code for the Go agent: https://github.com/elastic/go-sysinfo/blob/master/providers/linux/memory_linux.go#L71

Comment (NielkS) [reply elasticapm/metrics/sets/cpu_linux.py]: Thanks for the pointer, will do.

**Pull request merged**